contrib/librespot.service

```diff
@@ -1,5 +1,6 @@
 [Unit]
-Description=Librespot
+Description=An open source Spotify client
+Documentation=https://github.com/librespot-org/librespot
```

Comment:
You might also add `Documentation=https://github.com/librespot-org/librespot/wiki/Options` as a new line after `Documentation=https://github.com/librespot-org/librespot`

`Documentation` can be specified multiple times. The spec reads:

> A space-separated list of URIs referencing documentation for this unit or its configuration. Accepted are only URIs of the types "http://", "https://", "file:", "info:", "man:". For more information about the syntax of these URIs, see uri(7). The URIs should be listed in order of relevance, starting with the most relevant. It is a good idea to first reference documentation that explains what the unit's purpose is, followed by how it is configured, followed by any other related documentation. This option may be specified more than once, in which case the specified list of URIs is merged. If the empty string is assigned to this option, the list is reset and all prior assignments will have no effect.

Reply:
Done!